src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/AttendanceService.java

```diff
@@ -130,4 +154,11 @@ private AttendanceDto toAttendanceDto(Attendance attendanceData) {
                 .caseNoteId(attendanceData.getCaseNoteId())
                 .build();
      }
+
+    private String getIepRescindedReason(final UpdateAttendanceDto newAttendance) {
+        if (newAttendance.getAttended())
+            return "IEP rescinded: attended";
+
+        return "IEP rescinded: " + newAttendance.getAbsentReason().toString();
+    }
```

Comment:
Could this be something like:
```
return "IEP rescinded: " + (newAttendance.getAttended() ? "attended" : newAttendance.getAbsentReason().toString())
``` not sure if it's sufficiently readable, but reduces the string duplication of "IEP rescinded: "

Reply:
I like it!